include/bzfsAPI.h

```diff
@@ -843,6 +847,58 @@ class BZF_API bz_SlashCommandEventData_V1 : public bz_EventData
   bz_ApiString message;
 };
 
+class BZF_API bz_PollStartEventData_V1 : public bz_EventData
+{
+public:
+  bz_PollStartEventData_V1() : bz_EventData(bz_ePollStartEvent) {}
+
+  int playerID;
+  bool allow;
+  bz_ApiString reason;
+
+  // 'kick', 'kill', 'ban', 'set', 'reset', or custom value from a plug-in
+  bz_ApiString pollAction;
+
+  // If it's a 'kick', 'kill' or 'ban', this will be the victim's callsign
+  // If it's a 'set', this will be e.g. '_mirror black'
+  // If it's a 'reset', this will be 'flags'
+  bz_ApiString pollTarget;
+
+  // This value will be set to the victim's IP if it's a 'kick' 'kill' or 'ban'
+  bz_ApiString pollValue;
```

Comment:
`pollValue` since then has been removed since it's only available on certain occasions and having overload values are a pain, as you said. Besides, the information is only an IP so that can be fetched/stored by the plug-in